usaspending_api/etl/elasticsearch_loader_helpers/delete_data.py

```diff
@@ -199,53 +289,73 @@ def delete_awards(client: Elasticsearch, config: dict) -> None:
         client (Elasticsearch): elasticsearch-dsl client for making calls to an ES cluster
         config (dict): collection of key-value pairs that encapsulates runtime arguments for this ES management task
 
-    Returns: None
+    Returns: Number of ES docs deleted in the index
     """
-    deleted_ids = _gather_deleted_ids(config)
+    deleted_tx_keys = _gather_deleted_transaction_keys(config)
     # While extracting unique award keys, the lookup is on transactions and must match against the unique transaction id
-    id_list = [{"key": deleted_id, "col": ES_TRANSACTIONS_UNIQUE_KEY_FIELD} for deleted_id in deleted_ids]
-    award_ids = _lookup_deleted_award_ids(client, id_list, config, settings.ES_TRANSACTIONS_QUERY_ALIAS_PREFIX + "-*")
-    if (len(award_ids)) == 0:
+    award_keys = _lookup_deleted_award_keys(
+        client,
+        ES_TRANSACTIONS_UNIQUE_KEY_FIELD,
+        [*deleted_tx_keys],
+        config,
+        settings.ES_TRANSACTIONS_QUERY_ALIAS_PREFIX + "-*",
+    )
+    if (len(award_keys)) == 0:
         logger.info(format_log(f"No related awards require deletion", action="Delete"))
```

Comment:
Either this log statement or LN 309 should be updated so at least one of them is different to help when reading the logs

Reply:
I concur that'll be a helpful distinction. Changing